dist/index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;600;700;800&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="css/main.css" />
+  <link rel="stylesheet" href="css/media.css">
+  <title>Folio</title>
+</head>
+<body>
+
+  <!-- Burger content -->
+  <div class="burger-content">
+      <div class="avatar">
+        <img src="img/graph_shapes/shape1_small.png" alt="" class="small-shape avatar-lightning">
+        <img src="img/graph_shapes/shape3_small.png" alt="" class="small-shape  avatar-triple-wave">
+        <img src="img/graph_shapes/shape2.png" alt="" class="small-shape avatar-arrow">
+        <div class="download">
+          <button class="btn-resume">
+            <img src="img/topbar/download.png" alt="">
+          </button>
+          <p class="p-resume">DOWNLOAD RESUME</p>
+        </div>
+      </div>
+      <h1 class="greetengs">Hello, i’m <span class="span-name">John</span> <span class="span-surname">Smith</span></h1>
+      <p class="text-about">nec ullamcorper augue sagittis. Proin faucibus lorem nisl, Donec commodo leo est, quis eleifend urna pellentesque eget.</p>
+      <p class="text-about contact">Proin scelerisque odio eu nisi viverra, nec ullamcorper gue
+        <a href="mailto:contactme@mysite.com" class="contact-me">contact me</a></p>
+      <div class="skills">
+        <div class="skill-item photoshop">
+          <h4>PHOTOSHOP</h4>
+          <div class="progress">
+            <span class="span-skill" style="width: 98%; background: #c62fc6;"></span>
+          </div>
+          <p>98%</p>
+        </div>
+        <div class="skill-item illustrator">
+          <h4>ILLUSTRATOR</h4>
+          <div class="progress">
+            <span class="span-skill" style="width: 70%; background: #f2d25d;"></span>
+          </div>
+          <p>70%</p>
+        </div>
+        <div class="skill-item html-css">
+          <h4>HTML / CSS</h4>
+          <div class="progress">
+            <span class="span-skill" style="width: 60%; background: #2959e1;""></span>
+          </div>
+          <p>60%</p>
+        </div>
+      </div>
+      <div class="experience">
+        <div class="experience-item">
+          <h2>2000 - 2008</h2>
```

Comment:
this information doesn't contain anything that should be shown as h2

Reply:
h2 changed to <p class=".p-years">